src/core/object/pipeline/pipeline.ts

```diff
@@ -4,13 +4,21 @@
 import * as UserInput from '@/core/object/pipeline/user-input';
 import * as Metadata from '@/core/object/pipeline/metadata';
 import * as Normalize from '@/core/object/pipeline/normalize';
+import * as RegexEdits from '@/core/object/pipeline/regex-edits';
 import * as CoverArtArchive from '@/core/object/pipeline/coverartarchive/coverartarchive';
 import Song from '@/core/object/song';
 import { ConnectorMeta } from '@/core/connectors';
 
 export default class Pipeline {
 	private song: Song | null = null;
-	private processors = [Normalize, UserInput, Metadata, CoverArtArchive];
+	private processors = [
+		Normalize,
+		UserInput,
+		RegexEdits,
+		Metadata,
+		RegexEdits, // Run regex edits again, as the regex edit might have caused an album to be found.
```

Comment:
Why do we need to do it 2x vs just applying the regex edit at the end?

Reply:
I'm not 100% sure just naively running entire regexedits pipeline twice is the optimal solution, but we do need some form of regex edit application both before and after for expected behavior I think.

Reply:
Gonna mark this one as resolved after the regex pipeline edits from "Apply edits once per field"